packages/pages/src/upgrade/migrate_config.ts

```diff
@@ -12,33 +13,25 @@ function readJsonSync(jsonPath: string): any {
     console.error(`unable to parse ${jsonPath}: ${(e as Error).message}`);
     return null;
   }
-}
+};
 
-function writeYamlSync(configYamlPath: string, target: string, data: any) {
+const writeYamlSync = (configYamlPath: string, target: string, data: any) => {
   const yamlDoc = yaml.parseDocument(fs.readFileSync(configYamlPath, "utf-8"));
   yamlDoc.set(target, data);
   fs.writeFileSync(configYamlPath, yaml.stringify(yamlDoc));
-}
+};
 
-const migrateCiJson = async (
-  configYamlPath: string,
-  sitesConfigPath: string
-) => {
-  const ciJson = readJsonSync(path.resolve(sitesConfigPath, "ci.json"));
+const migrateCiJson = async (configYamlPath: string, ciPath: string) => {
+  const ciJson = readJsonSync(ciPath);
   if (ciJson !== null) {
-    ciJson.artifactStructure = null;
+    ciJson.artifactStructure = undefined;
```

Comment:
This isn't even a key. Just delete this line.

Reply:
This is actually necessary. Without it it will copy artifactStructure from ci.json